.travis.yml

```diff
@@ -17,6 +17,8 @@ env:
 
     - TEST_TYPE='integration' BLOCKCHAIN_TYPE='--blockchain-type=tester'
     - TEST_TYPE='integration' BLOCKCHAIN_TYPE='--blockchain-type=geth --blockchain-cache'
+    - TEST_TYPE='api' BLOCKCHAIN_TYPE='--blockchain-type=tester'
+    - TEST_TYPE='api' BLOCKCHAIN_TYPE='--blockchain-type=geth'
```

Comment:
No, should work fine with `--blockchain-cache`. In fact this is how I ran it locally. Let me include it.